rpm/sdk-setup.spec

```diff
@@ -302,6 +305,7 @@ fi
 %{_bindir}/sdk-setup-enginelan
 %{_bindir}/sdk-shutdown
 %{_bindir}/dynexecfs
+%{_bindir}/sdk-freespace
```

Comment:
I would prefer hiding it from PATH, so using `%{_libexecdir}/%{name}` as a prefix. Maybe we could even omit the executable and specify the oneliner literally in the .spec file as in `ExecStop=/bin/bash -c 'df ...'`?

Reply:
Right, specifying it directly on the service file makes sense as long as it stays oneliner.